test/lib/testMayaUsdPythonImport.py

```diff
@@ -32,5 +32,5 @@ def testImportModule(self):
         # mayaUsd library also loaded its dependencies (i.e. the core USD
         # libraries). We test the type name as a string to ensure that we're
         # not causing USD libraries to be loaded any other way.
-        stageCache = mayaUsdLib.StageCache.Get()
+        stageCache = mayaUsdLib.StageCache.Get(True)
```

Comment:
I believe this is the only python script that uses the StageCache python class I modified. There is also an Animal Logic version of the StageCache in python, most of the uses in the repo are their version that doesn't have a parameter.